en-US/Design.xml

```diff
@@ -282,23 +285,55 @@ $ vi myFile.txt
 
 				</example>
 				 <para>
-					If the commands are long, complex, or wrap over multiple lines, use the following design to help optimize clarity.
-          If you are documenting commands on a different operating system, update the prompts and line continuation characters to suit.
-				</para>
+					If the commands are long, complex, or wrap over multiple lines, two design options are available.
+          </para>
+          <itemizedlist>
+            <listitem>
+              <para>
+                Use line continuation characters and the associated PS2 prompts.
+                If you are documenting commands on a different operating system, update the prompts and line continuation characters to suit.
+              </para>
+            </listitem>
+            <listitem>
+              <para>
+                Use neither line continuation characters nor the associated PS2 prompts.
+              </para>
+            </listitem>
+          </itemizedlist>
+          <important>
+            <para>
+              Do not mix these two styles.
+              Maintain the same style throughout your document.
+            </para>
+          </important>
+          <para>
+            You can also indent the second and subsequent lines of such commands to assist in clarity and readability if required.
+            This option is available for either of the two designs mentioned above.
+          </para>
+          
 				 <example>
-					<title>Documenting Long Commands</title>
+					<title>Wrapping Long Commands with Continuation Characters</title>
+          <para>
+            This example uses both continuation characters and PS2 prompts.
+            These indicators are always used together.
+          </para>
 
 <screen># tar --selinux -czvf config_files.tar.gz  /etc/katello \
 > /etc/elasticsearch /etc/candlepin /etc/pulp /etc/gofer \
 > /etc/grinder /etc/pki/katello /etc/pki/pulp /etc/qpidd.conf \
 > /etc/sysconfig/katello /etc/sysconfig/elasticsearch \
-> /root/ssl–build /var/www/html/pub/* /var/lib/katello
+> /root/ssl–build /var/www/html/pub/* /var/lib/katello</screen>
+          </example>
 
-# cd /var/lib/katello
+<example><title>Wrapping Long Commands Without Continuation Characters</title>
+  <para>
+    This example uses neither continuation characters nor PS2 prompts, but it does demonstrate how to use line indentation to help clarify long commands.
```

Comment:
One of the examples in the userinput change I made included a paragraph of text explaining that the line was long and wrapped but that the user should ensure it is all one continuous line when they type it out:

![continuation](https://user-images.githubusercontent.com/52800133/100875086-81e3b780-3473-11eb-8322-2bd3525adbd2.png)

Is that language something that should be included with this example?

Reply:
Interesting question. I know we want to do this for commands that include URLs or class names, etc., that are long and have no spaces to provide for natural wrap points, but to do it for all(?) the commands that wrap would add a lot of content.
We're also considering adding a para somewhere near the beginning of the doc that describes how we use (or don't use) cont. chars and PS2 prompts, but that would happen only once.
I recommend not including this for now.